### PR TITLE
[IOPID-628] :  404 static error page

### PR DESCRIPTION
**Pull request by OraldoDoci**
## Short description

Created 404 error page because the default next-js page (not-found.js) does not generate the static page with output: export. Here is the link to the open issue on next: https://github.com/vercel/next.js/issues/54239. So we created the page in src/app/[locale]/(pages)/404/page.tsx

## List of changes proposed in this pull request

- New static 404 error page
- Changed href for the footer links

## How to test

- yarn run build
check that in the  folder out/it  exist the static 404.html page

-yarn run start-static
 go to /404

### CODE REVIEW

File: src/app/[locale]/(pages)/404/page.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import { Grid } from '@mui/material';
+import { IllusError } from '@pagopa/mui-italia';
+import { useTranslations } from 'next-intl';
+import { FeedbackMessage } from '../../_component/feedbackMessage/feedbackMessage';
+import { commonBackground } from '../../_utils/styles';
+
+const LoginKo = (): React.ReactElement => {
```
Comment (shadowsheep1) [inline src/app/[locale]/(pages)/404/page.tsx:9]: Why do we call this `LoginKo`?

File: src/app/[locale]/_component/footer/footer.tsx [hunk 2]
```diff
@@ -29,19 +29,19 @@ export default function Footer({
       links: [
```
Comment (shadowsheep1) [inline src/app/[locale]/_component/footer/footer.tsx:29]: Is this related to this PR?

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: src/app/[locale]/_component/footer/footer.tsx (hunk 2 above)
Comment (OraldoDoci) [reply src/app/[locale]/_component/footer/footer.tsx:29]: This was a fix that was added with this PR. I wrote it in the "List of changes proposed in this pull request" section as the second feature. There is no task in jira for this.

File: src/app/[locale]/(pages)/404/page.tsx (hunk 1 above)
Comment (OraldoDoci) [reply src/app/[locale]/(pages)/404/page.tsx]: Changed the name to NotFoundPage.

File: src/app/[locale]/_component/footer/footer.tsx (hunk 2 above)
Comment (shadowsheep1) [reply src/app/[locale]/_component/footer/footer.tsx:29]: Is this the fix for the javascript security warnings?

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: src/app/[locale]/(pages)/404/page.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <Grid sx={commonBackground} container>
+      <Grid item xs={12} justifySelf={'center'}>
+        <FeedbackMessage
+          topIcon={<IllusError />}
+          title={'Qui non c’è nulla!'}
+          summary={<span>Questa pagina non esiste o non c’è più.</span>}
+          firstButton={{
+            href: process.env.NEXT_PUBLIC_URL_IO,
```
[the remark this answers is not in the document]
Comment (OraldoDoci) [reply src/app/[locale]/(pages)/404/page.tsx]: Yes you are correct. When the PR was first made it only existed the figma for the anonymous session case. Updated the button text and link to be dynamic based on the user login status.

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**